src/components/streaming/StreamManagerItem/index.tsx

```diff
@@ -6,12 +6,14 @@ import { TokenIcon } from 'components/common/TokenIcon';
 import { blockInvalidChar } from 'utils/blockInvalidChars';
 import { calculateFlowRate } from 'utils/calculateFlowRate';
 import {
+	twoWayMarketibAlluoUSDETHAddress,
 	twoWayMarketDAIWETHAddress,
-	// twoWayMarketMATICDAIAddress,
+	twoWayMarketMATICDAIAddress,
 	twoWayMarketWBTCAddress,
 	twoWayWETHMarketAddress,
-	// twoWayMarketMATICUSDCAddress,
-	// twoWayMarketWBTCDAIAddress,
```

Comment:
@Seroxdesign at some point one of your PRs reverted these comments, idk what that might have broken, maybe the stream manager UI, Im uncommenting them as part of this PR :+1: